test/hotspot/jtreg/runtime/Thread/TestAlwaysPreTouchStacks.java

```diff
@@ -96,7 +96,6 @@ private static ReservedCommitted runPreTouchTest(boolean preTouch) throws Except
       Collections.addAll(vmArgs, "TestAlwaysPreTouchStacks", "test");
       ProcessBuilder pb = ProcessTools.createLimitedTestJavaProcessBuilder(vmArgs);
       OutputAnalyzer output = new OutputAnalyzer(pb.start());
-      output.reportDiagnosticSummary();
```

Comment:
IIUC by moving this into the exception throwing paths, it means that if `main` detects a failure and throws, there will not be any output reported. All you needed to do with this was move it after line 118:
```
   output.shouldMatch("- *Thread.*reserved.*committed");
```

Reply:
Sorry for the misunderstanding and thank you for the precise correction.
Now it is done.